### DISCUSSION TITLE
Icon - Lobe UI

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#67] [Q&A]
# Icon - Lobe UI

Icons are graphical symbols used to represent a concept or an action. The `Icon` component is a reusable React component that renders an SVG icon from the `lucide-react` library.

https://ui.lobehub.com/components/icon

**Comment by rdmclin2** [comment]
可以支持下 disabled

**Comment by scottdingzhi** [comment]
12334
